.coveragerc

```diff
@@ -451,6 +452,7 @@ omit =
     homeassistant/components/sensor/fritzbox_netmonitor.py
     homeassistant/components/sensor/gearbest.py
     homeassistant/components/sensor/geizhals.py
+    homeassistant/components/sensor/github.py
```

Comment:
This isn't a clean diff. Unrelated changes should not be included in a PR.

Reply:
Looks like this was github not showing the correct diff here in the PR. The merged commit has the correct clean diff. Sorry about that!